### PR TITLE
new esri leaflet react sample

### PR DESCRIPTION
**Pull request by jgravois**
- fixed links in my browserify sample
- added new [`React`](https://facebook.github.io/react/) sample

### CODE REVIEW

**Comment by goldenlimit** [comment]
Looks nice and I guess squash first and I can help to merge it

**Comment by jgravois** [comment]
@ungoldman.  does this have your blessing?

File: web-leaflet/react/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+1. fork/clone this github repository
+2. navigate into this folder using a command prompt and run `npm install`
+> this will lay down the project [dependencies](https://github.com/Esri/developer-support/blob/pages/web-leaflet/react/package.json#L8-L21) in the same folder.
+
+3. run `npm start`
+> afterward you'll be able to check out the sample application running at http://localhost:8080
+
+## Why React?
+
+Why not?  React must be cool.  Its an [open source project](https://facebook.github.io/react/) from Facebook developers.
```
Comment (ungoldman) [inline web-leaflet/react/README.md]: React is so 2015. It's all about [choo](https://github.com/yoshuawuyts/choo) now. 🚂🚋🚋🚋

**Comment by ungoldman** [comment]
@jgravois LGTM. This just an adaptation of https://github.com/ungoldman/github-user-map/tree/gh-pages/react right?

File: web-leaflet/react/README.md [hunk 2]
```diff
@@ -84,6 +84,10 @@ calling `npm run production` (only select keywords like `install` and `test` don
 
 check out [this great article](http://www.jayway.com/2014/03/28/running-scripts-with-npm/) on running scripts with npm.  [**@andersjanmyr**](https://github.com/andersjanmyr) does a far better job explaining than i ever could.
 
+## Credits
+
+* this work borrows substantially from @ungoldman's [github-user-map](https://github.com/ungoldman/github-user-map/tree/gh-pages/react) (with permission)
```
Comment (ungoldman) [inline web-leaflet/react/README.md]: Ah yeah I see. Permission granted.

File: web-leaflet/react/README.md (hunk 1 above)
Comment (jgravois) [inline web-leaflet/react/README.md]: i'll make sure we include a sample for it just as soon as its appeal falls past the apex. :stuck_out_tongue_winking_eye:

**Comment by jgravois** [comment]
> Looks nice and I guess squash first and I can help to merge it

there is no need to ask folks that submit a PR to squash their commits anymore. 
the ✨ [GitHub UI](https://github.com/blog/2141-squash-your-commits)✨  supports this feature now.

**Comment by jgravois** [comment]
as i said on 8/15, this PR doesn't need to be squashed prior to merging.  it can be done using GitHub's handy new 'Squash and merge' button.

**Pull request closed without merging**